Gemfile

```diff
@@ -47,4 +47,5 @@ group :test do
   gem 'launchy'
   gem 'poltergeist'
   gem 'database_cleaner'
+  gem 'factory_girl_rails', '~> 4.0'
```

Comment:
I'm not calling you a liar :) .. sometimes pinning has to happen to avoid conflicts.. but where does it say it in the readme? https://github.com/thoughtbot/factory_girl_rails